blueocean-github-pipeline/pom.xml

```diff
@@ -43,6 +51,11 @@
         <artifactId>pubsub-light</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>fluent-hc</artifactId>
```

Comment:
So, you are deliberately bundling this JAR, whatever it is?

Reply:
yes we are using this library.